session-manager/Cargo.toml

```diff
@@ -22,7 +22,9 @@ std = [
 [dependencies]
 err-derive = "0.2"
 policy-utils = { path = "../policy-utils" }
-ring = "0.16.11"
-rustls = "0.16"
-webpki = "=0.21.2"
-webpki-roots = "0.19.0"
+ring = "0.16.20"
+rustls = "0.20.4"
+rustls-pemfile = "0.3.0"
+veracruz-utils = { path = "../veracruz-utils" }
+webpki = "=0.22"
```

Comment:
Does `webpki`'s version need to be pinned?  If so, can you add a comment in the `Cargo.toml` file indicating why?